src/Dropdown/Examples.md

```diff
@@ -151,6 +151,35 @@ const options = [
 />;
 ```
 
+#### Multiselect with "select all" option
+
+Using a `MultiDropdownWithSelectAll` component, it's possible to automatically add an "all" option that will select all the available options at once.
+
+```js
+intialState = {};
+const onChange = (value, isAll) => {
```

Comment:
I don't really like this API to check if `all` is selected. I need to play a little bit more with it